cmd/etcd-launcher/main.go

```diff
@@ -406,7 +422,8 @@ func (e *etcdCluster) isHealthyWithEndpoints(endpoints []string) (bool, error) {
 	_, err = client.Get(ctx, "healthy")
 	defer cancel()
 	if err != nil && err != rpctypes.ErrPermissionDenied {
-		return false, err
+		// wait.Poll() doesn't retry if we err out
```

Comment:
This comment assumes the caller of the function. A better comment would be agnostic to the caller and describe the intention, e.g. `// silently swallow/drop transient errors`. At least I think :-)